src/pkg/task/dao/execution.go

```diff
@@ -329,11 +329,18 @@ func (e *executionDAO) querySetter(ctx context.Context, query *q.Query) (orm.Que
 				break
 			}
 		}
-		if len(keyPrefix) == 0 {
+		if len(keyPrefix) == 0 || keyPrefix == key {
 			return qs, nil
 		}
-		inClause, err := orm.CreateInClause(ctx, "select id from execution where extra_attrs->>?=?",
-			strings.TrimPrefix(key, keyPrefix), value)
+
+		// key with keyPrefix supports multi-level query operator on PostgreSQL JSON data
+		keys := strings.Split(strings.TrimPrefix(key, keyPrefix), ".")
```

Comment:
give an example in the comments to better understand.

Reply:
> give an example in the comments for better understanding.

will add more examples for this.